src/Nethermind/Nethermind.Specs.Test/ChainSpecStyle/ChainSpecBasedSpecProviderTests.cs

```diff
@@ -777,4 +789,13 @@ void TestTransitions(ForkActivation activation, Action<ReleaseSpec> changes)
         });
         TestTransitions((40001L, 1000000024), r => { r.IsEip1153Enabled = true; });
     }
+
+    private static bool ValidateEthereumSlotByTimestamp(ulong timestamp, ulong genesisTimestamp) =>
+        ValidateSlotByTimestamp(timestamp, genesisTimestamp, 12UL);
+
+    private static bool ValidateGnosisSlotByTimestamp(ulong timestamp, ulong genesisTimestamp) =>
+        ValidateSlotByTimestamp(timestamp, genesisTimestamp, 5UL);
```

Comment:
very minor: Would drop those and just make `ValidateSlotByTimestamp's` `blockTime` default to 5s.